src/Compilers/Core/Portable/Operations/OperationKind.cs

```diff
@@ -170,10 +170,12 @@ public enum OperationKind
         //PointerIndirectionReferenceExpression = 0x204,
         /// <summary>Indicates an <see cref="IIsPatternExpression"/>.</summary>
         IsPatternExpression = 0x205,
-        /// <summary>Indicates an <see cref="IIncrementExpression"/>.</summary>
+        /// <summary>Indicates an <see cref="IIncrementExpression"/> for increment operator.</summary>
         IncrementExpression = 0x206,
         /// <summary>Indicates an <see cref="IThrowExpression"/>.</summary>
         ThrowExpression = 0x207,
+        /// <summary>Indicates an <see cref="IIncrementExpression"/> for decrement operator.</summary>
+        DecrementExpression = 0x208,
```

Comment:
Why is decrement an `IIncrementExpression`? Consider renaming `IIncrementExpression`, perhaps to `IIncrementOrDecrementExpression` if there is no better name.

Reply:
@dotnet/analyzer-ioperation - I like Chuck's suggestion. Any objections?